repose-aggregator/tests/performance-tests/Jenkinsfile

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env groovy
+
+properties([
+        buildDiscarder(logRotator(numToKeepStr: '14')),
+        pipelineTriggers([cron('H H(0-7) * * *')]),
+        parameters([string(
+                defaultValue: "",
+                description: "Optional - Extra options to add to the end of the ansible-playbook command.",
+                name: "extraOptions")])
+])
+
+stage("Performance Test") {
+    // current list of performance tests to run (with the optional scripting language if applicable)
+    def perfTestsWithScriptLang = [
```

Comment:
That's the thing I realized wouldn't be easy because of the security restrictions in the Jenkins sandbox. We wrote a spike to investigate: https://repose.atlassian.net/browse/REP-5574